ow_power_system/src/power_system_node.cpp

```diff
@@ -24,6 +24,10 @@ const string FAULT_NAME_THERMAL = "thermal_power_failure";
 //
 static constexpr int TEMPERATURE_INDEX = 1;
 
+// GLOBAL VARS (should implementation be changed to avoid this?)
+bool one_fault_injected = false;
```

Comment:
First, this needn't be a global variable, just a file variable.  Prefix a `static` to the declaration and remove the comment.

Second, per a (proposed) coding guideline on Confluence, a better name would be OneFaultInjected to help distinguish this variable from function variables and arguments.